services/core/shared/core/compat/sdk_v5/transactions.js

```diff
@@ -72,10 +72,11 @@ const getTransactionIndexingInfo = async (blocks) => {
 	let multisignatureInfoToIndex = [];
 	const publicKeysToIndex = [];
 	const recipientAddressesToIndex = [];
-	const txnMultiArray = blocks.map(block => {
+	const txnMultiArray = blocks.map((block, index) => {
 		const transactions = block.payload.map(tx => {
 			const [{ id }] = availableLiskModuleAssets
 				.filter(module => module.id === String(tx.moduleID).concat(':').concat(tx.assetID));
+			tx.index = index;
```

Comment:
why are we storing the index of block as index of transaction?

Reply:
Fixed!